packages/dapp-high-roller/src/components/app-game-status/app-game-status.scss

```diff
@@ -0,0 +1,45 @@
+.divider::before {
+  position: absolute;
+  content: "";
+  z-index: 0;
+}
+.divider {
+  position: relative;
+  width: calc(100% - 2.5rem);
+  margin: 2.5rem auto;
+  box-shadow: inset 0 -1px 0 0 #00d49f, inset 0 0 6px 0 rgba(0, 0, 0, 0.5);
```

Comment:
@Alonski You might want to use $variables to keep color definitions as consistent as possible.

Reply:
Same as above but possible.